### ISSUE TITLE
Regarding Implementation info

### ISSUE THREAD

**Issue by mrayush18** [sayvazov/AttentionNet#1]
Is this the implementation of [AttentionNet: Aggregating Weak Directions for Accurate Object Detection](https://arxiv.org/pdf/1506.07704.pdf) ?

**Comment by sayvazov** [comment]
No. This is an Attention Based Recurrent Neural Network designed for work
with time/frequency space audio data. If you want the source of that
project, I recommend emailing the authors.
